usr/share/rear/layout/prepare/GNU/Linux/110_include_lvm_code.sh

```diff
@@ -110,11 +110,16 @@ elif lvm vgcfgrestore --force -f "$VAR_DIR/layout/lvm/${vg}.cfg" $vg >&2 ; then
         [ \$vg == "$vg" ] || continue
         # Consider Thin Pools only
         [[ ,\$layout, == *,thin,* ]] && [[ ,\$layout, == *,pool,* ]] || continue
-        # Remove twice, the first time it fails because of thin volumes in the pool
-        lvm lvremove -q -f -y \$vg/\$lv || true
-        lvm lvremove -q -f -y \$vg/\$lv
+        # Use "--force" twice to bypass any error due to invalid transaction id
+        lvm lvremove -q -f -f -y $vg/\$lv
     done
 
+    # Once Thin pools have been removed, we can activate the VG
+    lvm vgchange --available y $vg >&2
+
+    LogPrint "Sleeping 3 seconds to let udev or systemd-udevd create their devices..."
+    sleep 3 >&2
```

Comment:
Not sure if you are aware of [retry_command()](https://github.com/rear/rear/blob/master/usr/share/rear/lib/layout-functions.sh#L652). Could it be somehow helpful in this case, instead of hard coding static value?
I'm asking because with [retry_command()](https://github.com/rear/rear/blob/master/usr/share/rear/lib/layout-functions.sh#L652) user can set [REAR_SLEEP_DELAY](https://github.com/rear/rear/blob/master/usr/share/rear/conf/default.conf#L874) and [REAR_MAX_RETRIES](https://github.com/rear/rear/blob/master/usr/share/rear/conf/default.conf#L880) ...

Reply:
Don't think it applies here because we do not know at all if there will be some LVs activated in that VG.